cartography/intel/aws/iam.py

```diff
@@ -168,7 +168,8 @@ def load_policies(session, policies, current_aws_account_id, aws_update_tag):
 def load_roles(session, roles, current_aws_account_id, aws_update_tag):
     ingest_role = """
     MERGE (rnode:AWSRole{arn: {Arn}})
```

Comment:
I would add the label with `MERGE rnode:AWSRole:AWSPrincipal{arn: {Arn}})` versus at the `on create set` so that all the labels for a node are defined in the same place.

Reply:
Wouldn't that result in a MATCH being done on all AWSPrincipal nodes rather than just AWSRole nodes though?

Reply:
Oh, good point, oops